globals/strings.gd

```diff
@@ -497,7 +520,9 @@ func build_choose_cards_string(from_zone, to_zone, amount_min, amount_max,
 	var card_str = tr("cards")
 	if amount_min == 1 and amount_max == 1:
 		card_str = tr("card")
-	var main_text = tr("Choose {AMOUNT} {CARD} {FROMZONE} to move to {TOZONE}{REMAIN}.").format({AMOUNT = amount_str, CARD = card_str, FROMZONE = from_zone_str, TOZONE = to_zone_str, REMAIN = remaining_cards_str})
+	var main_text = tr("Choose {AMOUNT} {CARD} {FROMZONE} to move to {TOZONE}{TOLIMITATION}{REMAIN}.")\
```

Comment:
If you change this, you need to change it in the translations.csv too

Reply:
Yeah. I caught this change an hour ago while working on something else. I've fixed it here 9841d98aa1c292a7978499a142a4567c7a54f386.